### PR TITLE
+cc65 inline asm stp mnemonic support

### PR DESCRIPTION
**Pull request by mlauke**
hi,

want to support stp mnemonic from within inline asm of a c program, to break into my emulator debugger.

cheers,

https://github.com/twoinke/steckschwein-emulator

### CODE REVIEW

File: src/cc65/codeent.c [hunk 1]
```diff
@@ -615,6 +615,9 @@ void CE_GenRegInfo (CodeEntry* E, RegContents* InputRegs)
         case OP65_BRK:
             break;
 
+        case OP65_STP:
+            break;
+
```
Comment (greg-king5) [inline src/cc65/codeent.c]: Please move that case to a position in alphabetical order.  That is, place it just above `case OP65_STX:`.

File: src/cc65/opcodes.h [hunk 2]
```diff
@@ -114,6 +114,7 @@ typedef enum {
     OP65_SED,
     OP65_SEI,
     OP65_STA,
+    OP65_STP, /* 65c02, 65816 stop */
```
Comment (greg-king5) [inline src/cc65/opcodes.h]: Please move the comment over to character column 33 (that is, align it with the other comments).

Comment (polluks) [reply src/cc65/opcodes.h]: Now jump on one leg ;-)

**Comment by oliverschmidt** [comment]
@mlauke: Thanks for contributing to cc65.

@greg-king5: Please merge as you see fit.

**Comment by mlauke** [comment]
jumped on one leg...

**Pull request merged**